src/utils/HTTPClient.ts

```diff
@@ -143,6 +143,10 @@ export default class HTTPClient {
         ctx.client.clientName = Constants.CLIENTS.YTMUSIC_ANDROID.NAME;
         ctx.client.androidSdkVersion = Constants.CLIENTS.ANDROID.SDK_VERSION;
         break;
+      case 'TV_EMBEDDED':
+        ctx.client.clientVersion = Constants.CLIENTS.TV_EMBEDDED.VERSION;
+        ctx.client.clientName = Constants.CLIENTS.TV_EMBEDDED.NAME;
```

Comment:
I think you should also add `ctx.client.clientScreen = "EMBED"` in order to be a little more compliant with the youtube servers and the name of the client used.
that's what we do at invidious: https://github.com/iv-org/invidious/blob/master/src/invidious/yt_backend/youtube_api.cr#L167

Reply:
Will do, thanks for the heads up!